setup.py

```diff
@@ -38,10 +38,13 @@
     test_suite='runtests.runtests',
     tests_require=[
         'pyjwkest>=1.3.0',
```

Comment:
I kept pyjwkest in the tests to show that this would all be compatible. Just want to show I'm not adding or removing functionality, just refactoring some libraries out.